.github/workflows/release.yml

```diff
@@ -57,4 +57,21 @@ jobs:
       - name: Upload Release Assets to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
\ No newline at end of file
+          password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
+
+  trigger_docs_pr:
+    name: Trigger schematisation checks PR workflow on threedi-docs
+    needs: [publish]
+    runs-on: ubuntu-latest
+    steps:
+      - name: install cURL
+        run: sudo apt update && sudo apt-get install -y curl
+      - name: send POST request to Github API
+        run: |
+          curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.ELI_TRIGGER_DOCS_TOKEN }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/elisalle/threedi-docs/actions/workflows/update-checks-list.yml/dispatches \
+          -d '{"ref":"master","inputs":{"reviewer_name":"${{ github.actor }}"}}'
```

Comment:
Netjes om 'em over losse lijnen te splitsen.
In de URL zit nog "elisalle", da's neem ik aan voor het testen nu?

Voor het testen zou je tijdelijk ook de "needs: publish" weg kunnen halen, dat lijkt me prima.

Reply:
> Netjes om 'em over losse lijnen te splitsen. In de URL zit nog "elisalle", da's neem ik aan voor het testen nu?

Oeps, die zat er nog in van het testen op een clone.

Reply:
> Voor het testen zou je tijdelijk ook de "needs: publish" weg kunnen halen, dat lijkt me prima.

Ik dacht eigenlijk straks gewoon een patch release te doen, maar dat kan ook.